recipes-kernel/linux/linux-tegra_4.9.bb

```diff
@@ -187,3 +189,12 @@ python () {
             bb.warn('did not find it in %s' % ','.join(flags))
             pass
 }
+
+SRCTREECOVEREDTASKS += "do_kernel_add_regdb"
+do_kernel_add_regdb() {
+    if ${@bb.utils.contains('DISTRO_FEATURES', 'wifi', 'true', 'false', d)}; then
+        cp ${STAGING_LIBDIR_NATIVE}/crda/db.txt ${S}/net/wireless/db.txt
+    fi
+}
+do_kernel_add_regdb[dirs] = "${S}"
+addtask kernel_add_regdb before do_build after do_configure
```

Comment:
That `before do_build` should be `before do_compile`, since the file is needed during kernel compilation (the bbclass in meta-networking has this wrong, too).

Reply:
Will update the recipe